plugins/ltac/g_tactic.mlg

```diff
@@ -53,7 +53,7 @@ let test_lpar_id_coloneq =
 
 (* Hack to recognize "(x)" *)
 let test_lpar_id_rpar =
-  Pcoq.Entry.of_parser "lpar_id_coloneq"
```

Comment:
Notice that the original code uses "lpar_id_coloneq" twice, which would overwrite that parser entry.  A bug.